Project/Assets/Prairie/Framework/Script/Interaction/Prompt.cs

```diff
@@ -4,16 +4,35 @@
 [AddComponentMenu("Prairie/Utility/Prompt")]
 public class Prompt : MonoBehaviour 
 {
-	public string promptText;
+    private static readonly int FIRST_PROMPT = 1;
+    private static readonly int SECOND_PROMPT = 2;
+
+    public bool isCyclic = false;
+
+    public int curPrompt = FIRST_PROMPT;
+    public string firstPrompt = "";
+    public string secondPrompt = "";
 	
+    public string GetPrompt()
+    {
+        if (curPrompt == FIRST_PROMPT)
+        {
```

Comment:
Just a code style thing - but this whole function could be reduced down to a single one line terinary operator.